src/bundle/Controller/ContentController.php

```diff
@@ -331,4 +333,57 @@ public function previewAction(
             'versionNo' => $versionNo ?? $content->getVersionInfo()->versionNo,
         ]);
     }
+
+    /**
+     * @param Request $request
+     *
+     * @return Response
+     *
+     * @throws AdminInvalidArgumentException
+     * @throws \InvalidArgumentException
+     * @throws TranslationInvalidArgumentException
+     * @throws APIRepositoryInvalidArgumentException
+     * @throws UnauthorizedException
+     * @throws InvalidOptionsException
+     */
+    public function updateMainTranslationAction(Request $request): Response
+    {
+        $form = $this->formFactory->updateMainTranslation();
+        $form->handleRequest($request);
+        if ($form->isSubmitted() && $form->isValid()) {
+            $result = $this->submitHandler->handle($form, function (MainTranslationUpdateData $data) {
+                $contentInfo = $data->getContentInfo();
+                $mapper = new MainTranslationUpdateMapper();
```

Comment:
In kernel it's called `main language` and I would use this name instead of `main translation` to avoid confusion.